packages/cli/src/start/server.ts

```diff
@@ -0,0 +1,199 @@
+import fs from 'node:fs'
+import path from 'node:path'
+
+import { type AnyModels, handleGetRequest } from '@rise-tools/server'
+import chokidar from 'chokidar'
+import Fastify from 'fastify'
+import internalIp from 'internal-ip'
+import qrcode from 'qrcode-terminal'
+import { WebSocketServer } from 'ws'
+import z from 'zod'
+
+// TODO
+import { connectWebSocket, createWSServerContext } from '../../../server/src/ws-connection'
```

Comment:
@ericvicenti can we export these from `@rise-tools/server`?

Reply:
I have a question about this here: https://github.com/rise-tools/rise-tools/pull/154#discussion_r1677033356